js/src/common/utils/Drawer.js

```diff
@@ -50,7 +74,7 @@ export default class Drawer {
 
     $app.removeClass('drawerOpen');
 
-    if (this.$backdrop) this.$backdrop.remove();
+    this.$backdrop.remove?.();
```

Comment:
Nice

Reply:
This changes the logic. You want to check for $backdrop, the current code checks that the `remove` button is truthy, which will error if `$backdrop` is undefined

![image](https://user-images.githubusercontent.com/6401250/129460003-8a86c7e7-ef30-43ab-9129-2d87e7020974.png)

Reply:
Now I look at it, why do we need this at all?

From what I can see, after you remove the element with jQuery, the variable doesn't get cleared.

The only scenario would be if `hide` was somehow called before `show`. Is that even possible?

```js
let x = $('<div />').appendTo('body')
!!x // true
x.remove()
!!x // true
x.remove()
// ...
```

Reply:
I'd still keep the check just in case. 🤷 No reason not to have it, right?

Reply:
True. I've combined the two as if the value is anything other than `undefined`, we get an error if we only have a `?` after `$backdrop.`